test/test_datasets_download.py

```diff
@@ -223,103 +184,84 @@ def root():
 
 
 def places365():
-    return itertools.chain(
-        *[
-            collect_download_configs(
-                lambda: datasets.Places365(ROOT, split=split, small=small, download=True),
-                name=f"Places365, {split}, {'small' if small else 'large'}",
-                file="places365",
+    return itertools.chain.from_iterable(
```

Comment:
Just a style nit: Instead of `itertools.chain(*[])` we now use `itertools.chain.from_iterable([])` which is made for this very specific use case.